ckui/src/ui/3delements/scripts/element_graphview_3D.gd

```diff
@@ -122,22 +123,23 @@ func end_streaming():
 	if root_node == null:
 		root_node = graph_data.nodes[ graph_data.nodes.keys()[0] ].scene
 	
-	var total_descendants := 0.0
-	var descendants_values := []
-	for node in graph_data.nodes.values():
-		if ("metadata" in node.data
-		and "descendant_count" in node.data.metadata
-		and (node.kind == "aws_account" or node.kind == "gcp_project")):
-			var node_descendant_count = node.data.metadata.descendant_count
-			descendants_values.append(node_descendant_count)
-			total_descendants += node_descendant_count
-	
-	if !descendants_values.empty():
-		var largest_descendant_value = descendants_values.max()
-		
+	if size_by_descendants:
+		var _total_descendants:= 0.0
+		var descendants_values := []
 		for node in graph_data.nodes.values():
-			if "metadata" in node.data and "descendant_count" in node.data.metadata:
-				node.scene.descendant_scale = node.data.metadata.descendant_count / largest_descendant_value
+			if ("metadata" in node.data
+			and "descendant_count" in node.data.metadata
+			and (node.kind == "aws_account" or node.kind == "gcp_project")):
```

Comment:
we probably need something more generic..

Reply:
That whole block is inactive at the moment so it won't cause any issues for now.
But I agree that a more generic approach would be better.

This should also be solvable by using `get_children_count()` with the new structure.